BookingsApi/BookingsApi.IntegrationTests/Api/V1/Hearings/BookNewHearingTests.cs

```diff
@@ -380,6 +380,62 @@ public async Task should_send_relevant_message_to_the_queue_when_a_new_participa
         _hearingIds.Add(response.Id);
     }
 
+    [Test]
+    public async Task should_send_relevant_message_to_the_queue_when_a_new_judge_added_to_existing_hearing_for_new_templates()
```

Comment:
just a few thoughts on this test:

- The test title suggests that it's for when a judge is added to an existing hearing, but the actual test is not adding a judge, it's adding a litigant in person
- It's calling the V1 APIs, shouldn't it be calling the V2 ones to align with the ticket?
- I think it's in the wrong file (BookNewHearingTests.cs) - I would suggest moving it to the test for the API that it's calling (UpdateHearingParticipants.cs). As mentioned above, this should be the V2 version of the test

Reply:
sorry this was committed by mistake. I have deleted it

Reply:
fair enough, but we should still add a test covering this scenario

Reply:
I thought this kind of test we will integrate with the test framework

Reply:
the automation tests? Personally I would suggest adding the test here, that way we can assert on the message that's sent

Reply:
test added